### ISSUE TITLE
Stuck on "More that one start branch"

### ISSUE THREAD

**Issue by simonc** [Shopify/git-chain#32]
Hi,

With a large work being developed on branch `A` I created a chain with branches `master` and `A`.

I then created a branch named `B` and added it to the chain.

I added commits to `B`, then merged `B` into `A`

I ran `git pull` on master and pruned merged branches.

Now I can't update the chain, I can't tear it down. I can't even list chains. All I get is an error stating

```
More that one start branch: ["A", "master"]
```

Any way I can unlock this, apart from nuking the git-chain config?

Thanks!

**Comment by philkuz** [comment]
all the config info is stored in .git/config -> use at your own risk

**Comment by simonc** [comment]
@philkuz Yeah I did nuke the config in the end, but it was more to document the issue as is becomes a blocker and is "hard" to get around 😬 Thanks for your reply! 😊🙏
